src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -269,15 +285,32 @@ private function createVariants(ProductInterface $product, array $options)
         /** @var ProductVariantInterface $productVariant */
         foreach ($product->getVariants() as $productVariant) {
             $productVariant->setAvailableOn($this->faker->dateTimeThisYear);
-            $productVariant->setPrice($this->faker->randomNumber(4));
             $productVariant->setCode(sprintf('%s-variant#%d', $options['code'], $i));
             $productVariant->setOnHand($this->faker->randomNumber(1));
 
+            foreach ($this->channelRepository->findAll() as $channel) {
+                $this->createChannelPricings($productVariant, $channel);
+            }
```

Comment:
I think that unfortunately we have two :/ These products are in order fixtures taken to order and due to order process we have to have prices specified for each channel (otherwise it would 💥 )